docs/source/markdown/podman-create.1.md

```diff
@@ -419,6 +419,12 @@ Logging driver specific options.  Used to set the path to the container log file
 
 `--log-opt path=/var/log/container/mycontainer.json`
 
+**--log-opt**=*tag*
+
+Specify a custom log tag for the container.  For example:
+
+`--log-opt tag="{{.ImageName}}"`
```

Comment:
What's the format? What are valid keys and values?

Reply:
specified in the man page